src/components/counter/counter.css

```diff
@@ -0,0 +1,33 @@
+.counter-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+/* Style for decrement button */
+.decrement {
+    padding: 10px 20px;
+    background-color: #007bff;
+    color: #fff;
+    border: none;
+    border-radius: 4px 0 0 4px;
+    font-size: 16px;
+    cursor: pointer;
+}
+
+/* Style for increment button */
+.increment {
```

Comment:
add this class in line 8, .decrement, .increment because most of the properties are same. only one is different.
So for that .increment { border-radius: 0 4px 4px 0 }

Like below
```
.decrement, .increment {
    padding: 10px 20px;
    background-color: #007bff;
    color: #fff;
    border: none;
    border-radius: 4px 0 0 4px;
    font-size: 16px;
    cursor: pointer;
}

.increment { border-radius: 0 4px 4px 0; } 
```

Reply:
Fixed it.